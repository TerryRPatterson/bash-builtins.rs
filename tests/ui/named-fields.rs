use bash_builtins::BuiltinOptions;

#[derive(BuiltinOptions)]
enum Opt {
    #[opt = 'a']
    A(String),

    #[opt = 'b']
    B { s: String }
}

fn main() {}
