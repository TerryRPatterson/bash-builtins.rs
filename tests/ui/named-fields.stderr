error: Named fields are not supported
 --> $DIR/named-fields.rs:8:5
  |
8 |     #[opt = 'b']
  |     ^
